Source/WebCore/css/CSSValueKeywords.in

```diff
@@ -530,6 +530,8 @@ contents
 grid
 inline-grid
 flow-root
+math
+block-math
```

Comment:
I think `block-math` is not a valid CSS keyword so it shouldn't be added here (and you didn't add it to `Source/WebCore/css/CSSProperties.json`)